assets/js/plugin-check-admin.js

```diff
@@ -378,36 +378,63 @@
 			Date.now().toString( 36 ) +
 			Math.random().toString( 36 ).substr( 2 );
 
+		// Check if any errors or warnings have links.
+		const hasLinks =
+			hasLinksInResults( errors ) || hasLinksInResults( warnings );
+
 		// Render the file table.
 		resultsContainer.innerHTML += renderTemplate(
 			'plugin-check-results-table',
-			{ file, index }
+			{ file, index, hasLinks }
 		);
 		const resultsTable = document.getElementById(
 			'plugin-check__results-body-' + index
 		);
 
 		// Render results to the table.
-		renderResultRows( 'ERROR', errors, resultsTable );
-		renderResultRows( 'WARNING', warnings, resultsTable );
+		renderResultRows( 'ERROR', errors, resultsTable, hasLinks );
+		renderResultRows( 'WARNING', warnings, resultsTable, hasLinks );
 	}
 
 	/**
-	 * Renders a result row onto the file table.
+	 * Checks if there are any links in the results object.
 	 *
 	 * @since n.e.x.t
 	 *
-	 * @param {string} type    The result type. Either ERROR or WARNING.
 	 * @param {Object} results The results object.
-	 * @param {Object} table   The HTML table to append a result row to.
+	 * @return {boolean} True if there are links, false otherwise.
+	 */
+	function hasLinksInResults( results ) {
+		for ( const line in results ) {
+			for ( const column in results[ line ] ) {
+				for ( let i = 0; i < results[ line ][ column ].length; i++ ) {
+					if ( results[ line ][ column ][ i ].link ) {
```

Comment:
I'm not familiar with this data structure, but it could perhaps be made less verbose/repetitious with `for`/`of` loops. Not important however.

Reply:
Not sure this could be optimized. Of course we could consider using nested array functions, but I think that would be more of a personal preference that wouldn't make this code faster or slower.

Reply:
For example:

```suggestion
		for ( const lineResult of results ) {
			for ( const columnResult of lineResult ) {
				for ( const result of columnResult ) {
					if ( columnResult.link ) {
```

Not really an optimization, but more a reduction in verbosity. 

The names of my variables could surely be improved.

Reply:
Thank you for the feedback. This code change will result in the exclusion of certain code sniff results, even if the code contains the necessary link.